schema/bom-1.2-strict.schema.json

```diff
@@ -27,14 +26,12 @@
       ]
     },
     "specVersion": {
-      "$id": "#/properties/specVersion",
```

Comment:
`$id` is the same value the JSON pointer has. so the `$id` is shadowing the exact same element and can therefore be removed.